File: Java/Somatorio.java
public class Somatorio {
    public static void main(String[] args) {
        int n = 10;
        int soma = 0;

        for (int i = 1; i <= n; i++) {
            soma += i;
        }

        System.out.println("O somatorio de 1 a " + n + " e: " + soma);
    }
}

File: JavaScript/Somatorio.js
function somatorio(n) {
    if (n < 0) {
        return "Número inválido. Por favor, insira um número não negativo.";
    }
    let soma = 0;
    for (let i = 1; i <= n; i++) {
        soma += i;
    }
    return soma;
}

console.log(somatorio(5));
